docs/pages/identity-governance/entra-id/terraform.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title:  Configure Entra ID integration using Terraform
+description: Describes how to set up Teleport Entra ID integration using Terraform
+sidebar_position: 3
+labels:
+ - how-to
+ - identity-governance
+ - terraform
+---
+
+This guide shows Entra ID integration configuration using Terraform and `tctl`.
```

Comment:
I would provide some context for when a user would want to follow this guide. Is this an all-Terraform alternative to the guided flow?